### PR TITLE
Adds organization page

### PR DESCRIPTION
**Pull request by bryclee**

### CODE REVIEW

File: public/scripts/admin/dashboard.jsx [hunk 1]
```diff
@@ -2,45 +2,62 @@
...
+            name: data.name,
+            logo: data.logo || null,
+            welcome_message: data.welcome_message || null
+          };
+        } else {
+          this.transitionTo('addOrg');
+        }
+        if (data.members) {
+          state.members = data.members.sort(function(a, b) {
+            return a.last_name.toUpperCase().charAt(0) > b.last_name.toUpperCase().charAt(0) ?
```
Comment (jpei) [inline public/scripts/admin/dashboard.jsx]: return a.last_name.toUpperCase() > b.last_name.toUpperCase() ? 1 : a.last_name.toUpperCase() < b.last_name.toUpperCase() ? -1 : a.first_name.toUpperCase() > b.last_name.toUpperCase() ? 1 : -1

File: public/scripts/admin/subcomponents/EditForm.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+var Router = require('react-router');
+
+// Renders the form displayed to edit a user
+var EditForm = React.createClass({
+  propTypes: {
+    members: React.PropTypes.array.isRequired
+  },
+  mixins: [Router.Navigation],
+  render: function() {
+    var id = parseInt(this.props.params.user, 10);
```
Comment (jpei) [inline public/scripts/admin/subcomponents/EditForm.jsx]: var id = +this.props.params.user;

File: public/scripts/shared/utils.jsx [hunk 3]
```diff
@@ -49,12 +63,13 @@ var parsePhone = function(phoneString) {
   if (!result || result.length !== 10) {
     return false;
   } else {
-    return parseInt(result, 10);
+    return parseInt(result.join(''), 10);
```
Comment (jpei) [inline public/scripts/shared/utils.jsx]: return +phoneString;

**Comment by bryclee** [comment]
The changes have been made.

**Comment by jpei** [comment]
Looks good

**Pull request merged**